src/args.py

```diff
@@ -39,9 +57,13 @@
 # key: foo, value: bar
 # key: baz, value: 12
 #
-# Google "python keyword arguments".
+# bing "python keyword arguments".
 
 # YOUR CODE HERE
+def f4(**stuffs):
+    for x in stuffs:
```

Comment:
Great Job! There's also the 'items()' method that returns a view object that displays a list of dictionary's (key, value) tuple pairs. you should check that one out.